runtimes/common/src/bonded_currencies/mod.rs

```diff
@@ -265,14 +269,14 @@ impl<
 
 	fn name(asset: Self::AssetId) -> Vec<u8> {
 		match Criterion::convert(asset) {
-			Left(()) => Vec::new(),
+			Left(()) => "KILT".as_bytes().to_vec(),
 			Right(a) => Right::name(a),
 		}
 	}
 
 	fn symbol(asset: Self::AssetId) -> Vec<u8> {
 		match Criterion::convert(asset) {
-			Left(()) => Vec::new(),
+			Left(()) => "KILT".as_bytes().to_vec(),
 			Right(a) => Right::symbol(a),
 		}
```

Comment:
I would like to hard code these values. Otherwise, I have to provide another generic to the type for having the different names for peregrine and spiritnet. Seems not worth to me.